amplify/backend/ts-code/src/api/BorrowItem.ts

```diff
@@ -46,19 +47,31 @@ export class BorrowItem {
      * @param borrower Name of borrower
      * @param notes Notes about this action
      */
-    public execute(scratch: ScratchInterface): Promise<string> {
+    public execute(input: BorrowItemInput): Promise<string> {
         return emitAPIMetrics(
             () => {
-                return Promise.all(scratch.ids.map((id: string) =>
-                    this.mainTable.changeBorrower(id, scratch.borrower, "borrow", scratch.notes)
-                )).then(() => `Successfully borrowed items '${scratch.ids.toString()}'.`)
+                return this.performAllFVAs(input)
+                    .then(()=>
+                        Promise.all(input.ids.map((id: string) =>
+                        this.mainTable.changeBorrower(id, input.borrower, "borrow", input.notes)
+                ))).then(() => `Successfully borrowed items '${input.ids.toString()}'.`)
```

Comment:
I would follow the indentation here: https://github.com/gp-seattle/rms-app/blob/master/amplify/backend/ts-code/src/api/ReturnItem.ts#L53-L58. Looks kinda messy.